include/sqlpp11/basic_expression_operators.h

```diff
@@ -124,11 +124,14 @@ namespace sqlpp
                                 wrap_operand_t<T>>;
 
     template <template <typename Lhs, typename... Rhs> class NewExpr, typename... T>
-    using _new_nary_expression_t =
+    struct _new_nary_expression
```

Comment:
Why aren't you using the inheritance technique here?

```
template <template <typename Lhs, typename... Rhs> class NewExpr, typename... T>
struct _new_nary_expression_t : new_nary_expression_t<logic::all_t<check_rhs_comparison_operand_t<ValueType, wrap_operand_t<T>>::value...>,
                                NewExpr,
                                Expr,
                                wrap_operand_t<T>...>;
```

That would require less changes now (and later when MSVC finally supports variadic template aliases in variadic templates).

Also, it would be really great, if you could add a comment with a reference to the MS Connect Bug number to each change so that it will be easier to revert them later.